docs/debug-kernel.md

```diff
@@ -0,0 +1,46 @@
+# Get cc-runtime Kernel logs
+
+The cc-runtime launches a virtual machine when a new pod is
+created. This VM uses a custom kernel inside the VM.
+
+The kernel boot logs are disabled by default. However these logs may be desired
+when the kernel is customized by the end user, or if the virtual machine boot
+fails due to the kernel.
+
+To provide a debug log of the kernel boot, Clear Containers supports an option
+to enable kernel boot logs using the configuration file.
+
+1. configuration.toml
```

Comment:
This should still read as: Enable debug in ..., but the file should be in literal text:
1. Enable debug in \`configuration.toml\`